Guide.docc/Swift6Mode.md

```diff
@@ -25,32 +25,35 @@ invocation using the `-Xswiftc` flag:
 
 ### Package manifest
 
-To change the language mode for package as a whole:
+A `Package.swift` file that uses `swift-tools-version` of `6.0` will enable
+the Swift 6 language mode for all targets.
+With that tools version, you can still change the language mode for the package
+as a whole:
 
 ```swift
-// swift-tools-version:6.0
+// swift-tools-version: 6.0
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 let package = Package(
-  name: "MyPackage",
-  products: [
-    // ...
-  ],
-  targets: [
-    // ...
-  ],
-  swiftLanguageVersions: [.v6]
+    name: "MyPackage",
+    products: [
+        // ...
+    ],
+    targets: [
+        // ...
+    ],
+    swiftLanguageVersions: [.v5]
 )
 ```
 
-To change the language mode on a per-target basis:
+You can also change the language mode on a per-target basis:
```

Comment:
Could make this one then about overriding to 6.0 instead, eg.
> And then migrate on a per-target basis using the new `swiftLanguageVersion` build setting:
> <what's there already>

Reply:
I went though a few different iterations here, and in the end, I settled on the simplest thing I think is most-likely to be helpful. What do you think?

Reply:
LGTM, thanks for iterating on it @mattmassicotte!